src/arch/xtensa/boot_loader.c

```diff
@@ -219,14 +219,17 @@ static uint32_t hp_sram_init(void)
 
 #endif
 
-#if defined(CONFIG_ICELAKE)
+#if defined(CONFIG_ICELAKE) || defined(CONFIG_APOLLOLAKE)
 static int32_t lp_sram_init(void)
 {
 	int status;
 	unsigned int timeout_counter, delay_count = 256;
 	timeout_counter = delay_count;
 
 	shim_write(SHIM_LDOCTL, SHIM_LDOCTL_LPSRAM_LDO_ON);
+#if defined(CONFIG_APOLLO_LAKE)
+	shim_write(SHIM_LSPGCTL, PLATFORM_LPSRAM_EBB_COUNT);
```

Comment:
PLATFORM_LPSRAM_EBB_COUNT has value 2.
You need LPSRAM_MASK here.

Reply:
Right.